kolibri_explore_plugin/assets/src/views/CustomChannelPresentationApp.vue

```diff
@@ -1,12 +1,18 @@
 <template>
 
-  <iframe
-    ref="iframe"
-    class="custom-presentation-iframe"
-    frameBorder="0"
-    :src="rooturl"
-  >
-  </iframe>
+  <b-overlay :show="isLoading">
```

Comment:
I wonder if wrapping the iframe will be problematic. The default is to wrap the overlay. But there is also a no-wrap option. Have you played with it? https://bootstrap-vue.org/docs/components/overlay#non-wrapping-mode

Reply:
I've tested with `noWrap` and it works correctly, maybe we can use that one. I've updated the PR to use the "non-wrapping-mode".